.zuul/playbooks/run.yaml

```diff
@@ -0,0 +1,44 @@
+- hosts: all
+  become: yes
+  roles:
+  - role: config-golang
+  - role: export-cloud-openrc
+    vars:
+      cloud_name: 'citynetwork'
```

Comment:
```
  roles:
  - config-golang
  - role: export-cloud-openrc
    vars:
      cloud_name: 'citynetwork'
```

Reply:
Ah sorry. I thought it doesn't make a difference so I wrote it like this for consistency reasons